src/unit_tests/wazuh_db/test_wdb_global.c

```diff
@@ -22,7 +22,7 @@ static int test_setup(void **state) {
     test_struct_t *init_data = NULL;
     os_calloc(1,sizeof(test_struct_t),init_data);
     os_calloc(1,sizeof(wdb_t),init_data->wdb);
-    os_strdup("000",init_data->wdb->id);
+    os_strdup("global.db",init_data->wdb->id);
```

Comment:
Even when any string works here, to be completely aligned with the real name we should use **global** instead of **global.db**

Reply:
FIXED